### ISSUE TITLE
Have better result in V1, V2 is too sharp

### ISSUE THREAD

**Issue by GOZARCK** [TencentARC/GFPGAN#48]
Have better result in faces with V1 model looks more natural. In  V2 is too sharp, but I don't have the color problem and the final result is more homogeneous throughout the image
![testV2model](https://user-images.githubusercontent.com/14808854/130144947-8df8ac38-70df-48fb-9e18-11224024d6c6.png)

**Comment by xinntao** [comment]
Personally, I think V2Model is better=-=

As V2Model also adds sharpening, so it may look sharper.

**Comment by GOZARCK** [comment]
>>V2Model are too white?
Yes,  it looks whiter than normal, and also the face, hair looks whiter too.
thanks!

**Comment by shubhamcodez** [comment]
> Personally, I think V2Model is better=-=
> By saying V1 model looks more natural, did you mean that the eyes in V2Model are too white?
> 
> As V2Model also adds sharpening, so it may look sharper.

Skin looks much smoother in V2, V1 gives out better texture. 
That's the major difference I can see. 
I think you reduced/ optimized input noises in V2, causing this.

**Comment by CongYuSanXun** [comment]
> 个人觉得V2Model比较好=-=
> 说V1模型看起来更自然，是不是说V2Model的眼睛太白了？
> 
> 由于 V2Model 还增加了锐化，所以它可能看起来更锐利。
请问如何可以去除锐化效果？

**Comment by SkyePeabody** [comment]
May I ask how you installed the V1 model, I can't install it anyway

**Comment by kevinrex66** [comment]
这个v1 model具体是哪个版本？
